routes.js

```diff
@@ -7,7 +7,7 @@ function fastifySwagger (fastify, opts, next) {
     url: '/',
     method: 'GET',
     schema: { hide: true },
-    handler: (request, reply) => reply.redirect(`.${opts.prefix}/`)
+    handler: (request, reply) => reply.redirect(`${opts.prefix}/index.html`)
```

Comment:
I don't think that `opts.prefix` here is correct. Can you add a test that loads this within another plugin, so that this is exposed as `/v1/documentation`? I think it should use [`fastify.basePath`](https://github.com/fastify/fastify/blob/master/docs/Server-Methods.md#basepath).

Reply:
nice catch!! fixed